db/migrate/20211109185240_create_monsters.rb

```diff
@@ -0,0 +1,14 @@
+class CreateMonsters < ActiveRecord::Migration[6.1]
+  def change
+    create_table :monsters do |t|
+      t.string :name, null: false
+      t.string :habitat
+      t.text :description, null: false
+      t.string :image, null: false
+      t.string :classification
+      t.integer :votes, default: 0
+      
+      t.timestamps
```

Comment:
Noticing some issues from other PR's as it looks like alot of branches may have been built of other feature branches. I would sync and look at the existing PR's and decide what needs to be fixed, reviewed and merged first to make things less complicated.  Let me know if you have any questions or need any help after syncing